index.html

```diff
@@ -17,184 +18,224 @@
     <header>
         <h2 class="logo">My logo</h2>
         <img src="images/Union.png" alt="hamburger">
+        <ul>
+            <li>Portfolio</li>
+            <li>About</li>
+            <li>Contact</li>
+        </ul>
     </header>
 
+
     <section class="header-section">
```

Comment:
- [ ] The background image of the d**esktop version** should match the original website, Kindly fix that.
- Your website 
![sdcsd](https://user-images.githubusercontent.com/26767383/154563028-d3367b49-4f22-494e-b05b-793781849b73.PNG)

- Original website
![Capture](https://user-images.githubusercontent.com/26767383/154562984-9f7b17cf-43ad-49f1-974e-65c33990fe9d.PNG)